utils/json-utils/src/main/scala/pl/touk/nussknacker/engine/json/JsonSchemaBuilder.scala

```diff
@@ -15,7 +15,6 @@ object JsonSchemaBuilder {
     SchemaLoader
       .builder()
       .useDefaults(useDefaults)
-      .nullableSupport(nullableSupport)
```

Comment:
please add note to MigrationGuide about this. Maybe also point that it concerns only RequestResponse as in Kafka, CRegistryClient handles parsing schema?

Reply:
I can add some info, but now is a little bit too late? Don't you think so?

Reply:
Why? We remove this now

Reply:
But this function does nothing :)

Reply:
No it does. When `nullableSupport` set, RequestResponse does not work with union schemas with null.

Reply:
As we have spoken, I'll change it on next pr.